ui/userguide.html

```diff
@@ -559,7 +559,7 @@
               <table id="queryparams">
                 <tr>
                   <td><span class="queryparam">nojobs</span></td>
-                  <td>Load result sets and revisions without loading any job results.</td>
+                  <td>Load pushes and revisions without loading any job results.</td>
```

Comment:
I'd like another opinion on changing this one. Can this be rephrased instead of just s/result sets/pushes/?